superset/views.py

```diff
@@ -1104,6 +1104,38 @@ def ping():
     return "OK"
 
 
+class KV(BaseSupersetView):
+
+    """Used for storing and retrieving key value pairs"""
+
+    @log_this
+    @expose("/store/", methods=['POST', 'GET'])
+    def store(self):
+        value = request.form.get('data')
+        if not value:
+            return Response("Error: data can not be empty", status=404)
+        base_url = request.form.get('baseUrl') \
+            if request.form.get('baseUrl') else ''
+        obj = models.KeyValue(value=value)
+        db.session.add(obj)
+        db.session.commit()
+        return("http://{request.headers[Host]}/{base_url}?id={obj.id}".format(
```

Comment:
flask nows base urls, we can pass view functions here and then use url_for(function_name, **kwargs)